editors/code/src/client.ts

```diff
@@ -119,19 +120,28 @@ export async function createClient(
                         ?.rendered;
                     if (rendered) {
                         if (preview) {
-                            const index = rendered.match(/^(note|help):/m)?.index || 0;
+                            const index =
+                                rendered.match(/^  = (note|help):/m)?.index || rendered.length;
```

Comment:
Why exactly was this changed? The regex is meant to stop at the first `note:`/`help:` section, there is no ` = ` in front of those

Reply:
Hmm, can you give an example of what are you trying to filter out?
I though it was this lines:
```rust
error[E0308]: mismatched types
 --> src/main.rs:3:9
  |
2 |     let x = 1;
  |             - expected due to this value
3 |     x = vec![1];
  |         ^^^^^^^ expected integer, found struct `Vec`
  |
  = note: expected type `{integer}`
           found struct `Vec<{integer}>`
  = note: this error originates in the macro `vec` (in Nightly builds, run with -Z macro-backtrace for more info)
```
which have `=` in front.

Reply:
```
error[E0038]: the trait `X` cannot be made into an object
 --> src\main.rs:2:12
  |
2 |     let _: Box<dyn X>;
  |            ^^^^^^^^^^ `X` cannot be made into an object
  |
note: for a trait to be "object safe" it needs to allow building a vtable to allow the call to be resolvable dynamically; for more information visit <https://doc.rust-lang.org/reference/items/traits.html#object-safety>
 --> src\main.rs:6:8
  |
5 | trait X {
  |       - this trait cannot be made into an object...
6 |     fn foo() -> Self;
  |        ^^^ ...because associated function `foo` has no `self` parameter
help: consider turning `foo` into a method by giving it a `&self` argument
  |
6 |     fn foo(&self) -> Self;
  |            +++++
help: alternatively, consider constraining `foo` so it does not apply to trait objects
  |
6 |     fn foo() -> Self where Self: Sized;
  |                      +++++++++++++++++


```

Reply:
Thanks,
I changed the regex back.